css/src/tokens/themes.scss

```diff
@@ -261,6 +265,8 @@ $themes: (
 		'primary-active': $palette-yellow-high-contrast-hover,
 		'primary-box-shadow': $palette-white,
 		'primary-invert': $palette-black,
+		'primary-hover-light-background': $palette-yellow-high-contrast-hover,
+		'primary-hover-light-text': $palette-black,
```

Comment:
`primary-hover-light-text` is the same as primary-dark except for high contrast. This was added in order to keep the tag hover text color the same as Atlas's current colors in high contrast. Alternatively, we update so that dark and high contrast also matches Fluent.

Reply:
Renamed to `primary-dark-hover`